instrumentation/servlet/servlet-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v2_2/Servlet2Advice.java

```diff
@@ -24,14 +26,20 @@ public static void onEnter(
       @Advice.Argument(value = 1, typing = Assigner.Typing.DYNAMIC) ServletResponse response,
       @Advice.Local("otelContext") Context context,
       @Advice.Local("otelScope") Scope scope) {
+    CallDepthThreadLocalMap.incrementCallDepth(Servlet2Advice.class);
 
     if (!(request instanceof HttpServletRequest) || !(response instanceof HttpServletResponse)) {
       return;
     }
 
     HttpServletRequest httpServletRequest = (HttpServletRequest) request;
 
-    if (tracer().getServerContext(httpServletRequest) != null) {
+    Context serverContext = tracer().getServerContext(httpServletRequest);
+    if (serverContext != null) {
+      // We're interested only in the very first suggested name, as this is where the initial
```

Comment:
Shouldn't this comment go to `updateServerSpanNameOnce` method declaration site?